my_game.py

```diff
@@ -371,19 +390,204 @@ def on_draw(self):
+
+        self.change_player_thrust_key_button = arcade.gui.UIFlatButton(
+            text="Change Thrust Key: " + self.id_to_key[CONFIG["PLAYER_THRUST_KEY"]],
+            width=300
+        )
+        self.v_box.add(self.change_player_thrust_key_button.with_space_around(bottom=20))
+
+        self.change_player_fire_key_button = arcade.gui.UIFlatButton(
+            text="Change Fire Key: " + self.id_to_key[CONFIG["PLAYER_FIRE_KEY"]],
+            width=300
+        )
+        self.v_box.add(self.change_player_fire_key_button.with_space_around(bottom=20))
+
+        self.change_player_turn_left_key_button = arcade.gui.UIFlatButton(
+            text="Change Turn Left Key: " + self.id_to_key[CONFIG["PLAYER_TURN_LEFT_KEY"]],
+            width=300
+        )
+        self.v_box.add(self.change_player_turn_left_key_button.with_space_around(bottom=20))
+
+        self.change_player_turn_right_key_button = arcade.gui.UIFlatButton(
+            text="Change Turn Right Key: " + self.id_to_key[CONFIG["PLAYER_TURN_RIGHT_KEY"]],
+            width=300
+        )
+        self.v_box.add(self.change_player_turn_right_key_button.with_space_around(bottom=20))
+
+        self.v_box.add(self.settings_guide_select.with_space_around(bottom=20))
+        self.v_box.add(self.settings_guide_exit.with_space_around(bottom=20))
+        self.v_box.add(self.settings_guide_key_already_in_use.with_space_around(bottom=20))
+
+        # Assign click functions to buttons
+        self.reset_settings_button.on_click = self.on_click_reset
+        self.change_player_thrust_key_button.on_click = self.on_click_change_keybind
+        self.change_player_fire_key_button.on_click = self.on_click_change_keybind
+        self.change_player_turn_left_key_button.on_click = self.on_click_change_keybind
+        self.change_player_turn_right_key_button.on_click = self.on_click_change_keybind
+
+        # Background Color
+        arcade.set_background_color(SCREEN_COLOR)
+        
+        # Add layout to UI Manager
+        self.manager.add(
+            arcade.gui.UIAnchorWidget(
+                anchor_x="center_x",
+                anchor_y="center_y",
+                child=self.v_box)
+        )
+
+    def on_click_change_keybind(self, event):
+        button_to_key_name = {self.change_player_thrust_key_button: "PLAYER_THRUST_KEY",
+            self.change_player_fire_key_button: "PLAYER_FIRE_KEY",
+            self.change_player_turn_left_key_button: "PLAYER_TURN_LEFT_KEY",
+            self.change_player_turn_right_key_button: "PLAYER_TURN_RIGHT_KEY"
+        }
+        self.name_of_key_to_change = button_to_key_name[event.source]
+        # The source of the UIEvent is the button that was clicked
+        event.source.text = self.settings_guide_press_key
+        self.settings_guide_select.text = ""
+
+    def on_click_reset(self, event):
+        # Reset CONFIG dict and delete user settings file if it exists
+        try:
+            os.remove("user_settings.toml")
```

Comment:
Please do not delete the file. Instead, rename it to _user_settings.toml.bak_. If that file exists, add an integer to the end of the filename until such a file does not exist.